repositories/PageRepository.test.js

```diff
@@ -440,4 +446,73 @@ describe("PagesRepository", () => {
       expect(position).toBe(pages[2].position);
     });
   });
+
+  describe("Duplicating a page", () => {
+    it("copies the necessary data into the duplicate from the original", async () => {
+      const { section } = await setup();
+
+      const page = buildPage({ sectionId: section.id });
+      const result = await PageRepository.insert(page);
+
+      const duplicatePage = await PageRepository.copyPage(result.id, 1);
+
+      const fieldsToOmit = ["id", "order", "title", "created_at", "updated_at"];
+
+      expect(omit(result, fieldsToOmit)).toMatchObject(
+        omit(duplicatePage, fieldsToOmit)
+      );
+    });
+
+    it("prepends 'Copy of' to the question title of the copy when duplicating a page", async () => {
+      const { section } = await setup();
+
+      const page = buildPage({ sectionId: section.id });
+      const result = await PageRepository.insert(page);
+
+      const duplicatePage = await PageRepository.copyPage(result.id, 1);
+
+      expect(duplicatePage.title).toContain("Copy of");
+    });
+
+    it("is able to insert the copied page below the original", async () => {
+      const { section } = await setup();
+
+      const pageOne = await PageRepository.insert(
+        buildPage({ sectionId: section.id })
+      );
+
+      const pageTwo = await PageRepository.copyPage(pageOne.id, 1);
+
+      const positionOfPageOne = await PageRepository.getPosition(pageOne);
+      const positionOfPageTwo = await PageRepository.getPosition(pageTwo);
+
+      expect(parseInt(positionOfPageOne) + 1).toEqual(
```

Comment:
It's a bit annoying that we have to use `parseInt` on the `position` values all the time.  I have created a Tech Debt ticket to investigate this: https://trello.com/c/r1NbxAwr/312-review-bigint-usage-in-api

Reply:
@adammahmood Alright. I'll resolve this conversation for now and then we can revisit this once the TechDebt card has been completed, if you agree? I don't feel that it should be a blocker for this task.

Reply:
I agree.